tests/sentry/tasks/integrations/github/test_patch_parsers.py

```diff
@@ -20,3 +20,254 @@ def test_python_in_class(self):
             "delete",
             "put",
         }
+
+    def test_javascript_simple(self):
+        patch = """@@44,38@@ function hello(argument1, argument2)
+
+@@44,38@@ export default function world({argument}) {
+
+@@44,38@@ export const blue = (argument) => {
```

Comment:
Arrow functions don't always have a curly brace

Reply:
added a case

Reply:
You can also have arrow functions like `const doThing = arg => arg + 1`